{
  "modified": "2024-10-16T14:43:09Z",
  "published": "2024-10-16T14:43:09Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2024-5316"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "libidi"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/libidi/MAL-2024-5316.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "c6e94cebe36aa568a3d86fd9d9d3ec154dd34f5f3a25d8144d31d2a235331e21",
        "import_time": "2024-10-24T00:59:23.982603603Z",
        "id": "RLUA-2024-08462",
        "modified_time": "2024-10-16T14:43:09Z"
      }
    ]
  }
}
